pulser/tests/test_simulation.py

```diff
@@ -729,31 +731,34 @@ def test_xy_mask_equals_remove():
     reg_two = Register({"q0": (0, 0), "q1": (10, 10)})
     pulse = Pulse.ConstantPulse(100, 10, 0, 0)
 
-    # Masked simulation
-    seq_masked = Sequence(reg_three, MockDevice)
-    seq_masked.set_magnetic_field(0, 1.0, 0.0)
-    seq_masked.declare_channel("ch_masked", "mw_global")
-    masked_qubits = ["q2"]
-    seq_masked.config_slm_mask(masked_qubits)
-    seq_masked.add(pulse, "ch_masked")
-    sim_masked = Simulation(seq_masked)
-
-    # Simulation on reduced register
-    seq_two = Sequence(reg_two, MockDevice)
-    seq_two.set_magnetic_field(0, 1.0, 0.0)
-    seq_two.declare_channel("ch_two", "mw_global")
-    seq_two.add(pulse, "ch_two")
-    sim_two = Simulation(seq_two)
-
-    # Check equality
-    for t in sim_two.evaluation_times:
-        ham_masked = sim_masked.get_hamiltonian(t)
-        ham_two = sim_two.get_hamiltonian(t)
-        assert ham_masked == qutip.tensor(ham_two, qutip.qeye(2))
+    for channel_type in ["mw_global", "rydberg_global", "raman_global"]:
+        # Masked simulation
+        seq_masked = Sequence(reg_three, MockDevice)
+        if channel_type == "mw_global":
+            seq_masked.set_magnetic_field(0, 1.0, 0.0)
+        seq_masked.declare_channel("ch_masked", channel_type)
+        masked_qubits = ["q2"]
+        seq_masked.config_slm_mask(masked_qubits)
+        seq_masked.add(pulse, "ch_masked")
+        sim_masked = Simulation(seq_masked)
+
+        # Simulation on reduced register
+        seq_two = Sequence(reg_two, MockDevice)
+        if channel_type == "mw_global":
+            seq_two.set_magnetic_field(0, 1.0, 0.0)
+        seq_two.declare_channel("ch_two", channel_type)
+        seq_two.add(pulse, "ch_two")
+        sim_two = Simulation(seq_two)
+
+        # Check equality
+        for t in sim_two._times:
```

Comment:
The failing tests come from the a PR on `develop` that got rid of the `_times` attribute. It was replaced by `sampling_times`, so it should just be a matter of doing
```suggestion
        for t in sim_two.sampling_times:
```

Reply:
Great, I also changed to `.sampling_times` on the test `test_mask_two_pulses()`